Mathlib/RepresentationTheory/GroupCohomology/Hilbert90.lean

```diff
@@ -24,15 +24,16 @@ Noether's generalization also holds for infinite Galois extensions.
 
 ## Main statements
 
-* `hilbert90`: for all $f: Aut_K(L) \to L^\times$ satisfying the 1-cocycle condition, there exists
-`β : Lˣ` such that $f(g)g(β) = β$ for all `g : Aut_K(L)`.
-* `groupCohomology.hilbert90`: $H^1(Aut_K(L), L^\times)$ is trivial.
+* `groupCohomology.isMulOneCoboundary_of_isMulOneCocycle_of_aut_to_units`: for all
```

Comment:
It looks like all mentions of Hilbert90 have left this file... please mention it in comments/docstrings.

Reply:
done, thanks!